### DISCUSSION TITLE
Can I Add a new programe language by myself?

### DISCUSSION THREAD

**Discussion by yangmijun** [massCodeIO/massCode#462] [Q&A]
I've noticed that there are some niche programming languages, such as VBA, that aren't included in the 160 programming languages, how can I added a new programming language support by myself?

**Comment by pa-0** [comment]
You can use asp.net (vb.net) to get proper syntax highlighting for VBA.  I know this doesn't answer your whole question (e.g. other niche languages), but it's a workaround / quick fix that you can at least use for VBA temporarily while you search for a better solution.
